backend/src/Edna/DB/Connection.hs

```diff
@@ -63,3 +67,42 @@ createConnPool (ConnString connStr) maxConnsNum = liftIO $ ConnPool <$>
 -- | Destroys a @ConnPool@.
 destroyConnPool :: MonadIO m => ConnPool -> m ()
 destroyConnPool (ConnPool pool) = liftIO $ destroyAllResources pool
+
+-- | Provides methods for working with connections to Postgres database.
+data PostgresConn = PostgresConn
+  { _withConnection  :: forall (m :: Type -> Type) a. MonadUnliftIO m
```

Comment:
According to our general convention I think it's better to call it `_pcWithConnection`. Also is `_` necessary? I think it's usually used for somewhat private things or when there is another thing with the same name without `_`. But here these things don't look private in any sense and apparently don't conflict with anything (especially if we add `pc` prefix).

UPD: I see that `_withConnection` conflicts with `withConnection` and in this sense it's somewhat private indeed. So maybe `_` is justified, but I still think we can just replace it with `pc` prefix.

Reply:
I'll do it